core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -831,6 +831,63 @@ public void second() {
     assertEquals("second(DATETIME '2020-08-17 01:02:03')", expression.toString());
   }
 
+  public void testSecondOfMinute(FunctionExpression dateExpression, int second, String testExpr) {
```

Comment:
Seems like both this method and `testInvalidSecondOfMinute` could be `private`.

Reply:
Fixed in 2b2f8b8b3873a049fe4c401c47a7eaaa71db154f